charts/umbrella/values.yaml

```diff
@@ -641,3 +641,21 @@ dataprovider:
     nameOverride: dataprovider-submodelserver
     ingress:
       enabled: false
+
+semantic-hub:
+  enabled: true
```

Comment:
as this `Jena Fuseki DB` image needs to be build first, please disable semantic-hub by default.

Reply:
done